### DISCUSSION TITLE
LogJam

### DISCUSSION THREAD

**Discussion by shoeper** [lunasec-io/lunasec#266] [General Feedback]
Hi,

I just noticed, there is already another CVE that was called LogJam: CVE-2015-4000 (see https://confluence.atlassian.com/kb/logjam-cve-2015-4000-and-atlassian-products-755140917.html)

Best
Sven

(Referring to https://www.lunasec.io/docs/blog/log4j-zero-day/)

**Comment by breadchris** [comment]
@shoeper thanks for raising this issue! As you can tell by looking at the internet now, we were quick to make the change to `log4shell` to avoid any confusion.
